functions/zoom-webhook.py

```diff
@@ -101,6 +158,55 @@ def handler(event, context):
     }
 
 
+def update_zoom_status(zoom_event, payload, correlation_id):
+    mid = payload["object"]["id"]
+    uuid = payload["object"]["uuid"]
+
+    status = None
+    if zoom_event == "recording.started" or zoom_event == "recording.resumed":
+        status = ZoomStatus.RECORDING_IN_PROGRESS
+    elif zoom_event == "recording.paused":
+        status = ZoomStatus.RECORDING_PAUSED
+    elif zoom_event == "recording.stopped":
+        double_urlencoded_uuid = quote(quote(uuid, safe=""), safe="")
+        r = zoom_api_request(
+            f"/past_meetings/{double_urlencoded_uuid}",
+            ignore_failure=True,
+        )
+        if r.status_code == 404:
+            status = ZoomStatus.RECORDING_STOPPED
```

Comment:
Recording stopped can translate to two statuses. Either "recording_stopped" if the meeting is still going on, or "recording_processing" if the meeting has ended. 

Getting 404 on the "past_meetings" endpoint means that the meeting has not ended yet and so we should use the "recording_stopped" status rather than "recording_processing"